src/ex6/client/src/components/todo-element.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import { deleteItem, markItemAsOld, changeItemStatus } from "../item_client.js";
+import PropTypes from 'prop-types';
+
+export function TodoElement(props) {
```

Comment:
consider adding a TodoElementWithAnimation component wrapper (this means you might need to accept className as a prop [consider using classnames])

Reply:
maybe on the next iteration